app/decorators/controllers/solidus_jwt/spree/api/base_controller_decorator.rb

```diff
@@ -0,0 +1,39 @@
+module SolidusJwt
```

Comment:
Reading through the solidus_support documentation I think the idea is to have a structure like:

```ruby
# /app/decorators/controllers/spree/api/base_controller_decorator.rb
module Spree
  module Api
    module BaseControllerDecorator
    end
  end
end
```